### PR TITLE
Inntekt API skeleton

### PR DESCRIPTION
**Pull request by geiralund**
- Added ktor test library to verify the API.

### CODE REVIEW

File: src/test/kotlin/no/nav/dagpenger/inntekt/ApiTest.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.testing.handleRequest
+import io.ktor.server.testing.withTestApplication
+import java.math.BigDecimal
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class ApiTest {
+
+    @Test fun ` should be able to Http GET inntekt `() = withTestApplication(Application::main) {
```
Comment (geiralund) [inline src/test/kotlin/no/nav/dagpenger/inntekt/ApiTest.kt]: ktor-server-test-host ^ library.  See https://ktor.io/servers/testing.html#testengine 
A this do not start any real http server it is superfast

Comment (androa) [reply src/test/kotlin/no/nav/dagpenger/inntekt/ApiTest.kt]: Cooolio!

File: src/main/kotlin/no/nav/dagpenger/inntekt/Api.kt [hunk 2]
```diff
@@ -1,13 +1,62 @@
...
+import io.ktor.routing.get
+import io.ktor.routing.route
+import io.ktor.routing.routing
+import io.ktor.server.engine.embeddedServer
+import io.ktor.server.netty.Netty
+import java.math.BigDecimal
+import java.util.concurrent.TimeUnit
 
-    fun some(): Boolean = true
+class Api {
```
Comment (androa) [inline src/main/kotlin/no/nav/dagpenger/inntekt/Api.kt]: I think the plain main method approach like used in : https://github.com/navikt/helse-sparkel/blob/master/src/main/kotlin/no/nav/helse/App.kt#L54 is moar neat than the companion object.

Comment (geiralund) [reply src/main/kotlin/no/nav/dagpenger/inntekt/Api.kt]: oo, cool!

**Pull request merged**